examples/index.md

```diff
@@ -162,3 +162,292 @@ Content-Type: application/vnd.api+json
   }
 }
 ```
+
+## Error Objects
+
+Examples of how [error objects](http://jsonapi.org/format/#error-objects) work.
+
+### Responses to failed resource creation:
+
+Response with one error object, with attribute error:
+
+```http
+HTTP/1.1 422 Unprocessable Entity
+Content-Type: application/vnd.api+json
+
+{
+  "errors": [
+    {
+      "title":  "Invalid Attribute",
+      "source": { "pointer": "data/attributes/first-name" },
+      "detail": "First name must contain at least three characters."
+    }
+  ]
+}
+```
+
+Notice that the attribute `first-name` is dasherized.  Dasherized attributes are preferred
+over underscored.
+
+Notice also that the "detail" is specific to this occurrence of the problem,
+whereas the "title" is more generic.
+
+Notice that the "source" points to the associated attribute with the error.
+
+Response with one error object, with relationship error:
+
+```http
+HTTP/1.1 422 Unprocessable Entity
+Content-Type: application/vnd.api+json
+
+{
+  "errors": [
+    {
+      "title":  "Missing Association",
+      "source": { "pointer": "data/relationships/author" },
+      "detail": "Post author cannot be blank."
+    }
+  ]
+}
+```
+
+Notice that the "source" points to the associated relationship with the error.
+
+Response with one error, using a "code":
+
+```http
+HTTP/1.1 422 Unprocessable Entity
+Content-Type: application/vnd.api+json
+
+{
+  "errors": [
+    {
+      "title": "Adapter Error",
+      "code": "ds.errors.invalid-error-expects-json-api-format",
+      "detail": "The adapter rejected the commit because it was invalid"
```

Comment:
Sure.  FWIW, I got that example title/code/detail off of the ember-data project :).